semanticdb-javac/src/main/java/com/sourcegraph/semanticdb_javac/SemanticdbTaskListener.java

```diff
@@ -83,7 +83,9 @@ public void finished(TaskEvent e) {
   // exception, it just prints the location with an empty message.
   private void reportException(Throwable exception, TaskEvent e) {
     ByteArrayOutputStream baos = new ByteArrayOutputStream();
-    exception.printStackTrace(new PrintWriter(baos));
+    PrintWriter pw = new PrintWriter(baos);
+    exception.printStackTrace(pw);
+    pw.close();
```

Comment:
This line is responsible for not seeing the root cause error message.

I've seen this type of empty message multiple times in customer reports:

> [ERROR] /Users/mwang/code/mwang/scip-java-test/src/main/java/com/imc/mwang/Foo.java:[1] error: semanticdb-javac:

🤞 going forward we'll be able to get more meaningful error reports.